src/data/icons.yml

```diff
@@ -6,6 +6,7 @@ mono:
   "attachment": "icons/attachment"
   "check": "icons/check"
   "cloud-four": "icons/cloud-four"
+  "close": "icons/x"
```

Comment:
Please use the same key here as the key you would need to reference to use the icon (in this case, `x`). Or change the filename to match. That way we can look at the docs to know which key to use.